src/handler-builder.ts

```diff
@@ -47,13 +47,22 @@ import * as testLab from './providers/testLab';
 export class HandlerBuilder {
   constructor() {}
 
+  /**
+   * Create a handler for HTTPS events.
+  
+   * `onRequest` handles an HTTPS request and has the same signature as an Express app.
+   * `exports.myFunction = functions.handler.https.onRequest((req, res) => { ... })`
+   *  This is identical to writing:
```

Comment:
We may want to really explicitly clarify that we're comparing extensions handlers to "regular" function handlers, or maybe avoid the "identical" phrasing.

Even just changing "identical" to "similar" would make me feel better.  WDYT?

Reply:
For HTTP functions, they are actually identical, haha. We created "handler.https" for consistency purposes (so that the handler namespace would have all the same event providers), not because it's actually useful. So I'm not sure what to do here. Maybe we can just remove these sentences altogether, in case they cause more confusion?

Reply:
Got it, didn't know that.

I think it's probably best to just remove these sentences, if you're OK with that.

Reply:
Done!